DataFormats/Common/interface/RefToBaseVector.h

```diff
@@ -161,6 +161,12 @@ namespace edm {
     return holder_ ? holder_->size() : 0;
   }
 
+  template <class T>
+  inline void RefToBaseVector<T>::reserve(size_type n) {
+    if (holder_ != nullptr)
+      holder_->reserve(n);
```

Comment:
This means that `reserve()` in 
```cpp
RefToBaseVector<T> foo;
foo.reserve(N);
```
is a no-op, right?

(I'm guessing @VinInn's use case to be
```cpp
        edm::RefToBaseVector<reco::Track> trackRefs;
        // trackRefs.vectorHolder()->reserve(trackCollection.size());  NOT a good idea
```
in https://github.com/cms-sw/cmssw/pull/33462/files#diff-509b7e4ef9898b7d529699031d792fa528489903211d78de7eb1e088736427ccR804-R805
)

Reply:
my understanding is that the semantic of the class is to create the holder if not found...
So the proposed mod may be no-op....